python/requirements.txt

```diff
@@ -6,6 +6,7 @@
 #
```

Comment:
Can we avoid deleting things from this file? In order to get the arm build to work, I believe we should only have to change the requirements for the wheel which are in `setup.py` (under `setup_spec.install_requires`)

Reply:
requirements.txt isn't quite up to date with setup.py so this gets them more in sync. Part of this is because we install the wheel with no-index which means we assume all of the dependencies are installed, I change that part though but I still think having a requirements.txt that is installable on ARM is useful.